### DISCUSSION TITLE
Read/save text strings from/to file?

### DISCUSSION THREAD

**Discussion by uhuznaa** [Yue-Yang/ChatGPT-Siri#77] [Ideas]
If you're using this with another language it's easy to translate the text strings in the variables (like welcome-text, continue-text and so on) once, but you have to repeat this with every update. If the shortcut would try to read these strings from a text file first (and write the file if it doesn't exist) one could translate the strings there and from then on easily update the shortcut without having to go through it and translate all text strings again and again. 

Basically support an optional configuration file in iCloud Drive that survives updates of the shortcut itself.

PS: Also with the API key of course, as of now I have to edit the shortcut and add my API key with every update of the shortcut which is a bit of a bother. If I could just add it to a config file in my iCloud Drive updating the shortcut would be much more convenient. This is by the way one of the shortcomings of shortcuts.
